tests/conftest.py

```diff
@@ -20,24 +20,28 @@
 
 import pytest
 import tftest
+import yaml
+
+from .collectors import pytest_collect_file
+from .fixtures import plan_summary_fixture, plan_validator_fixture
```

Comment:
Are the new imports used here?

Reply:
They're not used, it was a hack to make pytest discover the fixtures and collectors. I've removed the imports and used [`pytest_plugins`](https://docs.pytest.org/en/6.2.x/reference.html#globalvar-pytest_plugins) instead. I also moved the old fixture to their own file.